### ISSUE TITLE
Support IPv6 Ping

### ISSUE THREAD

**Issue by digitalfiend64** [ansible-collections/ansible.netcommon#424]
SUMMARY
Support pinging IPv6 devices

ISSUE TYPE
Feature Idea, Feature Request
COMPONENT NAME
ansible.netcommon.net_ping

ADDITIONAL INFORMATION
Used to ping remote devices via module instead of using an xxxx_command module to send a raw command and having to parse and modify return data

(IPv6 support should be standard across all network modules)

**Comment by Qalthos** [comment]
`net_ping` as a module does not do anything. It has documentation, but only as a convenience. It makes no assumptions nor restrictions on the parameters, the output, or how the module acts in any way. It merely takes the parameters you provide, forwards them to the real module `<network_os>_ping`, and returns whatever results that module provides.

It also should probably be deprecated as all the other `net_*` modules are, but apparently this one was even more forgotten than the others.

In short, there is nothing that can be done here. I see you already opened a similar issue on cisco.ios, which I presume was your actual target. As such, I'm closing this issue - when the ios PR is merged, `net_ping` will magically support IPv6 with no changes to this collection.
